data/avro-schema/src/main/resources/avro/net/corda/data/virtualnode/VirtualNodeState.avsc

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "enum",
+  "name": "VirtualNodeState",
+  "namespace": "net.corda.data.virtualnode",
+  "doc": "Virtual Node state.",
```

Comment:
I don't think naming this `VirtualNodeState` is quite correct, it's the status of an operational characteristic of a virtual node, for example, in the future a user may request to put their vnode into "ROLLING_UPGRADE" at the REST endpoint, which would result in flowP2pOperationalStatus and vaultDbOperationalStatus set to INACTIVE, but the others ACTIVE.

Probably a better name - `VirtualNodeOperationalState`

But this is just a minor internal thing, I'm happy to approve as generally this change is a good idea.

Reply:
I have pushed a rename to `VirtualNodeOperationalState`.